src/Sulu/Bundle/AdminBundle/Resources/config/routing_api.yaml

```
@@ -40,3 +40,14 @@ sulu_admin.get_items:
     format: json
     requirements:
         _format: json|csv
+
+# Block IDs
+sulu_admin.post_block_ids:
+    path: '/block-ids.{_format}'
+    methods: GET
```

We are generating something, so this should not be a GET as a GET is Idempotent. Only `POST` and `PATCH` requests should be used for generating things.

```suggestion
    methods: POST
```